src/app/components/Curation/index.tsx

```diff
@@ -63,28 +66,74 @@ export default ({
   });
 
   const GridComponent = getGridComponent(componentName);
+  const environmentIsLive = isLive();
 
   const isFirstCuration = position === 0;
   const curationSubheading = title || topStoriesTitle;
   const id = idSanitiser(curationSubheading);
 
+  // extract the first summary as the basis for the msg banner and the billboard
+  const [firstSummary] = summaries;
+  const {
+    description,
+    link: summaryLink,
+    imageAlt,
+    imageUrl,
+    isLive: summaryIsLive,
+    title: linkText,
+  } = firstSummary || {};
+
   switch (componentName) {
     case NOT_SUPPORTED:
       return null;
+    case BILLBOARD: {
+      if (firstSummary) {
+        return environmentIsLive ? (
+          <MessageBanner
+            heading={title}
+            description={description}
+            link={summaryLink}
+            linkText={linkText}
+            image={imageUrl}
+            eventTrackingData={{
+              componentName: `message-banner-${nthCurationByStyleAndProminence}`,
+              detailedPlacement: `${position + 1}`,
+            }}
+          />
```

Comment:
More out of curiosity, but why do we return the MessageBanner component on Live for type `BILLBOARD` rather than `null`?

Reply:
It's because the app is currently supporting Message Banners with the same TIPO settings as a Billboard on Live currently. At the moment we want to roll out the Billboards on test with the previous settings on `visualProminence` and `visualStyle`.

We need to be able to add the Billboard whilst still maintaining that support on the app. Hopefully that answers your q? if not happy to discuss further. Hope i've explained that right!

Reply:
Ah ok that makes sense, thanks!